File: Assignments/Assignment_17/p4.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of row: \n");
        scanf("%d",&row);
        for(int i=1;i<=row;i++){

                int num=1;
                for(int j=1;j<=i;j++){

                        if(j%2==0){

                                printf("A\t");
                        }else{

                                printf("%d\t",num);
                        }num++;
                }printf("\n\n");
        }
}

File: JAVA/14.Number_System/p4.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter a number:");
		int num=sc.nextInt();
		int count=1;
		for(int i=1;i<=num;i++){
		
			count*=i;
		}
		System.out.println("Factorial of "+num+" is: "+count);
	}
}
